wiki/People/Beatmap_Nominators/en.md

```diff
@@ -180,7 +180,7 @@ The tables listed below indicate the game mode(s) in which each Beatmap Nominato
 | ::{ flag=KR }:: [Kim_GodSSI](https://osu.ppy.sh/users/2218047) | Korean |
 | ::{ flag=KR }:: [Lami-](https://osu.ppy.sh/users/16505281) | Korean |
 | ::{ flag=PH }:: [lenpai](https://osu.ppy.sh/users/5314573) | Filipino |
-| ::{ flag=CN }:: [MadBricktree](https://osu.ppy.sh/users/4790958) | Chinese |
+| ::{ flag=CN }:: [MadBricktree](https://osu.ppy.sh/users/4790958) | Korean |
```

Comment:
I'm 909% sure this is wrong. (CN pointing to Korean)

Reply:
No he is korean he was just in china when he made thw account and he is just stuck on chinese flag now

Reply:
https://github.com/ppy/osu-wiki/pull/9116#issuecomment-1500925504 As confirmed by the man himself (you can also see he does a bunch of mods in Korean, and gets very lost and confused when people mod his stuff in Chinese)